spec/spec_helper_acceptance.rb

```diff
@@ -66,10 +67,10 @@
 
     # Ensure test files don't exist before we run the test suite
     absent_files = 'file{["c:/services.txt","c:/process.txt","c:/try_success.txt","c:/catch_shouldntexist.txt","c:/try_shouldntexist.txt","c:/catch_success.txt"]: ensure => absent }'
-    apply_manifest_on(windows_agents,absent_files) if windows_agents.count > 0
+    apply_manifest_on(windows_agents, absent_files, :catch_failures => true) if windows_agents.count > 0
```

Comment:
I think this stays with `apply_manifest_on` because we're using `puppet apply` here outside of the actual tests and don't need to go through the master, right?

Reply:
Correct.  It's agent only so no need for testmode switcher.